src/Framework/BuildErrorEventArgs.cs

```diff
@@ -199,25 +243,30 @@ public string ProjectFile
         }
 
         /// <summary>
-        /// Line number of interest in associated file. 
+        /// Line number of interest in associated file.
         /// </summary>
         public int LineNumber => lineNumber;
 
         /// <summary>
-        /// Column number of interest in associated file. 
+        /// Column number of interest in associated file.
         /// </summary>
         public int ColumnNumber => columnNumber;
 
         /// <summary>
-        /// Ending line number of interest in associated file. 
+        /// Ending line number of interest in associated file.
         /// </summary>
         public int EndLineNumber => endLineNumber;
 
         /// <summary>
-        /// Ending column number of interest in associated file. 
+        /// Ending column number of interest in associated file.
         /// </summary>
         public int EndColumnNumber => endColumnNumber;
 
+        /// <summary>
+        /// A link pointing to more information about the error.
+        /// </summary>
+        public string HelpLink => helpLink;
```

Comment:
I would prefer this be named `Url` or `HelpUrl`, but it sounds like `HelpLink` is already in use at the project system layer (https://github.com/microsoft/msbuild/issues/4029#issuecomment-450045271), so it's probably better to be consistent. So 👍

Reply:
Yeah, I picked that name because it was mentioned in that issue. I can change it of course, but it made sense to me to use what was suggested, rather than re-inventing the wheel ;).